### DISCUSSION TITLE
[Enhancement] Cross Platform Biometric Authentication

### DISCUSSION THREAD

**Discussion by mhrastegari** [dotnet/maui#157] [General]
## Summary
i think MAUI can/should have a cross platform Fingerprint/Face ID Authentication because now all the platforms have them all plus It’s a pain to have to type your password on the app every time when your device has some better and easier security things.

## Platforms

UWP: Windows Hello (iris scanner, FingerPrint, Pin)
iOS: Touch ID/Face ID 
macOS: Touch ID (if touchBar available)
Android: fingerprint reader (if available)

**Comment by Happypig375** [comment]
This is in the scope of https://github.com/xamarin/Essentials, not Xamarin.Forms/MAUI.

**Comment by mhrastegari** [comment]
so 🤔 i don't see any MAUI Essentials GitHub
plus Xamarin.Essentials will be System.Device obviously and i think all will be in one place, not separate again!

**Comment by Happypig375** [comment]
Well, System.Devices and MAUI are two separate parts of .NET 6.

**Comment by jamesmontemagno** [comment]
These will be two separate libraries and you will be able to use Essentials (System.Devices) in a .NET MAUI app. It is on the backlog on Essentials -> https://github.com/xamarin/Essentials/issues/71

**Comment by mhrastegari** [comment]
thanks james ❤

**Comment by alex-jitbit** [comment]
The Essentials issue has been closed in August 2023 as "not planned".

So. We're not getting any biometrics then?
